app/src/main/java/com/kovacnet/waterlevel/StatsActivity.java

```diff
@@ -163,6 +163,31 @@ private void fillStats()
 
 
     }
+    class Motor{  
```

Comment:
Whole section would need some formatting

Reply:
yes my friend i will make the necessary changes in the future days!
Also, i can try and make an english version of the code.